src/NaturalSort/NaturalSort.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
I made the NaturalSort project a .NET standard project since all it was doing was string manipulation/ordering.